package.json

```diff
@@ -7,13 +7,17 @@
   },
   "main": "index.js",
   "scripts": {
+    "examples": "npm-run-all --serial examples:*",
+    "examples:microsoft": "./bin/fastatic examples/microsoft.github.io-master/ --dest dist/examples/microsoft/",
+    "examples:react": "./bin/fastatic examples/react-gh-pages/ --dest dist/examples/react/",
+    "examples:bootstrap": "./bin/fastatic examples/bootstrap-gh-pages/ --dest dist/examples/bootstrap/",
+    "postexamples:react": "node ./scripts/post-react.js",
```

Comment:
Just because it's used as post hook, doesn't mean the script has to have that as a name. Is there something more descriptive?